hack/release.toml

```diff
@@ -61,12 +61,13 @@ stargz-snapshotter extension is now supported as Talos System Extension.
         title = "Component Updates"
         description = """\
 * DRBD: 9.2.6
-* QEMU agent: v8.1.1
-* Tailscale: 1.50.0
+* QEMU agent: v8.1.3
+* Tailscale: 1.54.0
 * Xen Guest Utilities: 8.3.1
-* Linux Firmware: 20231030
+* Linux Firmware: 20231111
 * gasket driver: 09385d4
 * Intel ucode: 20231114
+* NVIDIA: 535.129.03
```

Comment:
should we add mdadm to the list? probably a different section

Reply:
added that